--- AntTweakBar/src/LoadOGLCore.h 2013-03-20 17:29:06.000000000 -0400
+++ src/LoadOGLCore.h 2014-03-12 02:13:12.000000000 -0400
@@ -146,7 +146,7 @@
// GL 1.4
ANT_GL_CORE_DECL(void, glBlendFuncSeparate, (GLenum sfactorRGB, GLenum dfactorRGB, GLenum sfactorAlpha, GLenum dfactorAlpha))
ANT_GL_CORE_DECL(void, glMultiDrawArrays, (GLenum mode, const GLint first, const GLsizei count, GLsizei primcount))
-ANT_GL_CORE_DECL(void, glMultiDrawElements, (GLenum mode, const GLsizei count, GLenum type, const GLvoid indices, GLsizei primcount))
+ANT_GL_CORE_DECL(void, glMultiDrawElements, (GLenum mode, const GLsizei count, GLenum type, const GLvoid const indices, GLsizei primcount)) // leb
ANT_GL_CORE_DECL(void, glPointParameterf, (GLenum pname, GLfloat param))
ANT_GL_CORE_DECL(void, glPointParameterfv, (GLenum pname, const GLfloat params))
ANT_GL_CORE_DECL(void, glPointParameteri, (GLenum pname, GLint param))
@@ -211,7 +211,7 @@
ANT_GL_CORE_DECL(GLboolean, glIsProgram, (GLuint program))
ANT_GL_CORE_DECL(GLboolean, glIsShader, (GLuint shader))
ANT_GL_CORE_DECL(void, glLinkProgram, (GLuint program))
-ANT_GL_CORE_DECL(void, glShaderSource, (GLuint shader, GLsizei count, const GLchar string, const GLint length))
+ANT_GL_CORE_DECL(void, glShaderSource, (GLuint shader, GLsizei count, const GLchar const string, const GLint *length)) // leb
ANT_GL_CORE_DECL(void, glUseProgram, (GLuint program))
ANT_GL_CORE_DECL(void, glUniform1f, (GLint location, GLfloat v0))
ANT_GL_CORE_DECL(void, glUniform2f, (GLint location, GLfloat v0, GLfloat v1))